encryptor/network/message.py

```diff
@@ -118,10 +110,22 @@ def of(
                 content_length=len(content),
                 content_type=content_type,
                 content_encoding=content_encoding,
+                filename=filename,
+                mode=mode,
             ),
             content,
         )
 
+    def write_to_file(self, keys_dir) -> None:
+        """Writing bytes to file. Destination directory is based on argument"""
+
+        files_dir: Path = keys_dir / "files"
+        files_dir.mkdir(exist_ok=True)
+        file_path: Path = files_dir / self.headers.filename
+        if file_path.exists():
+            os.remove(file_path)
+        file_path.write_bytes(self.content)
+
```

Comment:
This method should take `file_path` as an argument.

Reply:
done